3-UsingAPIs/Week2/homework/ex2-pokemonApp/index.html

```diff
@@ -6,6 +6,10 @@
     <title>Pokemon App</title>
   </head>
   <body>
+    <div>
+      <button id="button" type="submit">Get Pokemon</button>
+      <select id="select"></select>
+    </div>
```

Comment:
In this exercise, one of the requirements is:
> Create and append DOM elements using JavaScript only. 

We meed to create those elements in JavaScript